tests/integration/deepspeed/deepspeed-model.py

```diff
@@ -0,0 +1,62 @@
+from djl_python import Input, Output
```

Comment:
Why not use built-in deepspeed.py?

Reply:
need some customization on tests and also build batches.